core/src/main/scripts/migrate_db.py

```diff
@@ -171,6 +285,9 @@ def run_migration(db_version, sql_filename, connection, cursor):
         # skip sql comments
         if line.startswith('--') and len(line) > 2 and line[2].isspace():
             continue
+        # Add reference genome variable
+        if line.strip('\n') == "     (SELECT REFERENCE_GENOME_ID FROM reference_genome WHERE BUILD_NAME='GRCh37')":
+            line = line.replace('GRCh37', reference_genome)
```

Comment:
Perhaps there's a more subtle way to insert the reference genome in the SQL statement.